Cargo.toml

```diff
@@ -656,7 +658,7 @@ slog-scope = "4.4.0"
 slog-term = "2.9.1"
 socket2 = { version = "0.5.7", features = ["all"] }
 static_assertions = "1.1.0"
-strum = { version = "0.26.3", default-features = false }
+strum = { version = "0.26.3", features = [ "derive" ] }
```

Comment:
Was this intentional? I remember @rumenov changed the default-features to false in https://github.com/dfinity/ic/commit/839976182a0d1d7a500962e241f300a859b6c3b6.

Reply:
Seems to make no difference. When I add default-features = false, and do cargo build, Cargo.lock does not change. Do you want me to add default-features back?

Reply:
Yeah I guess if it works with what was there then there's no point in changing things.